src/mouse/default_gutter_handler.js

```diff
@@ -133,6 +133,7 @@ class GutterTooltip extends Tooltip {
     constructor(editor) {
         super(editor.container);
         this.editor = editor;
+        this.visibleToolTipRow;
```

Comment:
nitpick: in the rest of the file we don't capitalize the second T in Tooltip, so we might want to be consistent here as well